torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2629,12 +2666,12 @@ void addBiasForConv2dIfNone(Module& module) {
   }
 }
 
-void swapDeQuant(Block* block) {
+void quantizeGeneralOps(Block* block) {
   auto graph = block->owningGraph();
   for (Node* n : block->nodes()) {
     if (n->kind() == prim::If) {
       for (Block* subblock : n->blocks()) {
-        swapDeQuant(subblock);
+        quantizeGeneralOps(subblock);
```

Comment:
This function seems to be replicating dequant for all uses. Can we call this before `replicateDeQuant` so that we have a single function that replicates dequant operation?

Reply:
I think so, I can change insertDeQuantForAllUse to insertDeQuant here

Reply:
oh actually no, we can't swap dequant if the dequant is not replicated before